Assets/MixedRealityToolkit.SDK/StandardAssets/Textures/IconMRTKLogo.png.meta

```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
+guid: 54b6ac91c62269f4caf384ce7d155313
```

Comment:
Don't we already have MRTK logo image in Assets already?

Reply:
oh, this is cropped one with the logo part without the text.
![2019-07-26 13_02_14-Unity 2018 4 2f1 Personal - DefaultManagerScene unity - MRTK-Public-Microsoft - ](https://user-images.githubusercontent.com/13754172/61978262-a38e7e00-afa5-11e9-9c30-e1b7122c878f.png)